gradle-plugin-api/src/main/java/wtf/emulator/gmd/EwManagedDevice.java

```diff
@@ -0,0 +1,23 @@
+package wtf.emulator.gmd;
+
+import com.android.build.api.dsl.Device;
+import org.gradle.api.provider.Property;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.Optional;
+import wtf.emulator.GpuMode;
+
+public interface EwManagedDevice extends Device {
+
+  @Optional
+  @Input
+  Property<String> getDevice();
```

Comment:
we call this `model` everywhere else whereas a `device` is a combiantion of `model, version, gpu`. Rename to `getModel()`?